src/XrdPfc/XrdPfcTrace.hh

```diff
@@ -64,13 +64,19 @@
    if (XRD_TRACE What >= TRACE_ ## act) SYSTRACE(XRD_TRACE, 0, m_traceID, 0, \
        TRACE_STR_ ## act << x << " " << GetLocalPath())
 
+#define TRACEF_INT(act, x) \
+   if (XRD_TRACE What >= act) \
+   {static const char* t_what[]={"","error ","warning ","info ","debug ","dump "};\
```

Comment:
This will create a static variable for each call site of `TRACEF_INT`. I see that another macro already does this above, though. Should we have a simple declaration shared by all macros instead?

Reply:
Agreed, created the array in XrdPfc namespace.